src/Item_Devices.php

```diff
@@ -1244,7 +1253,10 @@ public static function addDevicesFromPOST($input)
                 Html::displayNotFoundError();
             }
             if ($item instanceof CommonDevice) {
-                if ($link = getItemForItemtype($item->getItem_DeviceType())) {
+                if (
+                    ($link = getItemForItemtype($item->getItem_DeviceType()))
+                    && ($link instanceof Item_Devices)
```

Comment:
Maybe not needed anymore with the previous recommendation of not using `getType`.